extensions/elation/actions/closeCareGap/closeCareGap.ts

```diff
@@ -0,0 +1,50 @@
+import { Category, validate, type Action } from '@awell-health/extensions-core'
+import { z } from 'zod'
+import { makeAPIClient } from '../../client'
+import { SettingsValidationSchema, type settings } from '../../settings'
+import { dataPoints, fields, FieldsValidationSchema } from './config'
+
+export const closeCareGap: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'closeCareGap',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Close care gap',
+  description: 'Close a care gap in Elation.',
+  fields,
+  previewable: true,
+  dataPoints,
+  onActivityCreated: async (payload, onComplete, onError): Promise<void> => {
```

Comment:
You haven't addressed the `onActivityCreated` vs. `onEvent` review comment yet. If that's intentional, feel free to call it out and resolve it.

Reply:
I have, see https://github.com/awell-health/awell-extensions/pull/536#discussion_r1886502673. This is intentional.